client/app/assets/less/inc/ant-variables.less

```diff
@@ -0,0 +1,29 @@
+/* --------------------------------------------------------
+    Colors
+-----------------------------------------------------------*/
+@primary-color:                        #2196F3;
+
+
+/* --------------------------------------------------------
+    Font
+-----------------------------------------------------------*/
+@redash-font:                          -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+@font-family-no-number:                @redash-font;
+@font-family:                          @redash-font;
+@code-family:                          @redash-font;
+@font-size-base:                       13px;
+
+
+/* --------------------------------------------------------
+    Typograpgy
+-----------------------------------------------------------*/
+@text-color:                           #595959;
```

Comment:
I didn't find a place where this is noticeable (lmk if you see this anywhere), but this creates a slight difference between `@text-color` for Ant styling and for the rest (`#595959` vs `#767676`).
 
I changed this color because it seemed better for Ant Checkbox labels and Buttons:

**Before**
![checkbox-before](https://user-images.githubusercontent.com/3356951/51283369-1acbcf80-19cf-11e9-8bda-e642033d739b.png)
**After**
![checkbox-after 1](https://user-images.githubusercontent.com/3356951/51283437-52d31280-19cf-11e9-9c51-a5819076c42f.png)

It's probably better to pick one of those and set in both scopes, but I let this open for discussion :slightly_smiling_face:

Reply:
#595959 looks more suitable to me.

Reply:
@gabrieldutra I've noticed form labels have `font-weight: 500` which doesn't look suitable for checkbox and radio button labels (like in this screenshot). Doesn't look like it's an Ant style. Do you know what's causing this?

Reply:
```less
// inc/form.less 
label {
    font-weight: 500;
}

// ~bootstrap/less/forms.less
label {
  display: inline-block;
  max-width: 100%; // Force IE8 to wrap long content (see https://github.com/twbs/bootstrap/issues/13141)
  margin-bottom: 5px;
  font-weight: bold; // <- This
}
```

Reply:
Replacing the first label selector with `font-weight: normal` fixes this.